docs/features/secrets.adoc

```diff
@@ -194,12 +194,10 @@ secrets:
     file: ./secrets/jcasc_vault
 ```
 
-==== Using environment variables
+==== Using environment variables (not recommended)
 
-Environment variables can be directly read by JCasC when loading configurations.
-Secrets can be also injected using an environment variables.
-Note that such approach implies security risks,
-because the environment variables can be read by 
+It is strongly recommended consider other option instead of using environment variables for secrets. Such approach implies *security risks*,
+because the environment variables can be read by
```

Comment:
```suggestion
Although it is supported, it is not recommended to inject secrets using environment variables.
Doing so is *unsafe* because the environment variables can be read by
```

Reply:
updated in dc38d6a